platform/core/flatbuffers_shared/setup_flatbuffers.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e    # Exit if any command fails
+
+echo '===> Setting up flatbuffers'
+
+FLATBUFFERS_VERSION=v2.0.0
+FLATBUFFERS_REPO=https://github.com/google/flatbuffers
+
+cd flatbuffers_shared
```

Comment:
Same as above... Again, please consider Makefile (or ever Cargo files, if Rust is able to handle general-purpose tasks, which it will probably do)